buckw

```diff
@@ -191,6 +191,12 @@ setupBuckRun ( ) {
     setupBuckBinary
 }
 
+# Do not pass go, do not collect $200 go straight to kill - Skip okbuck if we just want to kill the buck daemon.
+if [[ "kill" == "$1" ]];
```

Comment:
Can we move this to a method declaration? We can handle all the cases in '$1' explicitly then